13/umbraco-cms/reference/cache/imemberpartialviewcacheinvalidator.md

```diff
@@ -0,0 +1,35 @@
+# ICacheRefresher
+
+This section describes what IMemberPartialViewCacheInvalidator is, what it's default implementation does and how to customize it.
+
+## What is an IMemberPartialViewCacheInvalidator
+
+This interface is used to isolate the logic that needs to run to invalidate parts of the PartialView cache when a member is updated
+
+## Why do we need to partialy invalidate the partialView cache
+
+Since some of the razor templates might show data that is retrieved from a member object and those templates might be cached by using the partial caching mechanism (i.e. `@await Html.CachedPartialAsync("member",Model,TimeSpan.FromDays(1), cacheByMember:true)`), we need to remove those cached partials when a member is updated.
+
+## Where is it used
+
+This interface is called from the MemberCacheRefresher which is called every time a member is updated.
+
+## Details of the implementation
+
+When a razor template partial is cached trough `Html.CachedPartialAsync` and `cacheByMember` is set to `true`, the extension method will append the memberId of the currently logged in member and a marker (i.e. `-m1015-`) to the partialView chachekey.
```

Comment:
```suggestion
Razor template partials are cached through a call to `Html.CachedPartialAsync` with `cacheByMember` set to `true`. This will append the id of the currently logged in member with a marker to the partial view cache key.  For example, `-m1015-`. 
```